konrad/radiation/arts.py

```diff
@@ -197,29 +162,26 @@ def calc_lookup_table(self, filename=None, fnum=2 ** 15, wavenumber=None):
 
         # Setup the lookup table calculation
         self.ws.AtmFieldsAndParticleBulkPropFieldFromCompact()
-        self.ws.vmr_field.value = self.ws.vmr_field.value.clip(min=0.0)
+        self.ws.vmr_field.value = self.ws.vmr_field.value.value.clip(min=0.0)
         self.ws.atmfields_checkedCalc()
         self.ws.abs_lookupSetup(p_step=1.0)  # Do not refine p_grid
         self.ws.abs_t_pert = np.arange(-160, 61, 20)
 
         nls_idx = [
-            i for i, tag in enumerate(self.ws.abs_species.value) if "H2O" in tag[0]
-        ]
-        self.ws.abs_speciesSet(
-            abs_species=self.ws.abs_nls,
-            species=[", ".join(self.ws.abs_species.value[nls_idx[0]])],
-        )
+            i for i, tag in enumerate(self.ws.abs_species.value) if "H2O" in tag
+        ][0]
+        self.ws.abs_nls = [self.ws.abs_species.value[nls_idx]]
 
         self.ws.abs_nls_pert = np.array(
             [10 ** x for x in [-9, -7, -5, -3, -1, 0, 0.5, 1, 1.5, 2]]
         )
 
         # Run checks
-        self.ws.abs_xsec_agenda_checkedCalc()
+        self.ws.propmat_clearsky_agenda_checkedCalc()
         self.ws.lbl_checkedCalc()
 
         # Calculate actual lookup table.
-        self.ws.abs_lookupCalc()
+        self.ws.abs_lookupCalc(abs_species=self.ws.abs_species)
```

Comment:
`self.ws.abs_lookupCalc()`: `abs_species` Argument is not needed.

Reply:
I thought it crashed before 🤔 but maybe that was due to another error in the configuration. I will try it with the final run script and remove it